Source/Core/TranslationRotationScale.js

```diff
@@ -16,7 +16,7 @@ import Quaternion from './Quaternion.js';
      * @param {Quaternion} [rotation=Quaternion.IDENTITY] A {@link Quaternion} specifying the (x, y, z, w) rotation to apply to the node.
      * @param {Cartesian3} [scale=new Cartesian3(1.0, 1.0, 1.0)] A {@link Cartesian3} specifying the (x, y, z) scaling to apply to the node.
      */
-    var TranslationRotationScale = function(translation, rotation, scale) {
+    function TranslationRotationScale(translation, rotation, scale) {
```

Comment:
The rest is just proper function definitions so that the function name is actually correct (rather than a variable with an anonymous function)